js/src/api/rpc/parity/parity.js

```diff
@@ -17,6 +17,14 @@
 import { inAddress, inData, inHex, inNumber16, inOptions } from '../../format/input';
 import { outAccountInfo, outAddress, outChainStatus, outHistogram, outNumber, outPeers, outTransaction } from '../../format/output';
 
+function inAddresses (addresses) {
```

Comment:
Good point, I initially moved it to the top to get going (testing), but never moved it to the formatters as it should be. (Nor really major since it is the only place it is used, but still not consistent.)